### DISCUSSION TITLE
Pi stopped producing images

### DISCUSSION THREAD

**Discussion by LloydOS** [jekhokie/raspberry-noaa-v2#303] [Q&A] [answered]
Hi - System has been working well for some time, but all of a sudden stopped producing images.
While the passes are working correctly. System seems to have stopped producing images.
Have restarted a number of time and checked SDR device.
Any suggestions.

`pi@RPiWXsat:~ $ sudo rtl_test
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T tuner
Supported gain values (29): 0.0 0.9 1.4 2.7 3.7 7.7 8.7 12.5 14.4 15.7 16.6 19.7 20.7 22.9 25.4 28.0 29.7 32.8 33.8 36.4 37.2 38.6 40.2 42.1 43.4 43.9 44.5 48.0 49.6
[R82XX] PLL not locked!
Sampling at 2048000 S/s.

Info: This tool will continuously read from the device, and report if
samples get lost. If you observe no further output, everything is fine.

Reading samples in async mode...
Allocating 15 zero-copy buffers
lost at least 12 bytes`

**Comment by jekhokie** [comment]
@LloydOS what does the output of `atq` look like on your Pi device. Also, check the logs in `/var/log/raspberry-noaa-v2/output.log` for any clues.

**Reply by LloydOS** [reply]
I had decided to do a clean install and now the webserver will not run?
Also `atq`
Gives me
`pi@RPiWXsat:~ $ atq
10      Mon Mar 22 21:29:00 2021 a pi
11      Mon Mar 22 17:02:00 2021 a pi
14      Mon Mar 22 19:30:00 2021 a pi
9       Mon Mar 22 19:49:00 2021 a pi
12      Mon Mar 22 18:42:00 2021 a pi
8       Mon Mar 22 18:54:00 2021 a pi
13      Mon Mar 22 17:50:00 2021 a pi`

**Comment by jekhokie** [comment, marked as answer]
@LloydOS you are likely in the default mode where the webserver is running on port 443 (TLS secured) - check your config/settings.yml file for what configurations are set for the following:


```
enable_non_tls: false
web_port: 80
enable_tls: true
web_tls_port: 443
cert_valid_days: 365
```
